dev/aura.html

```diff
@@ -355,9 +405,61 @@
           <vaadin-avatar abbr="J" style="--vaadin-avatar-size: 1.25lh" slot="prefix" tabindex="-1"></vaadin-avatar>
           <span>John Smith</span>
         </vaadin-button>
-        <vaadin-button theme="tertiary" id="theme-editor-btn">
+        <vaadin-button theme="tertiary" id="theme-editor-btn" aria-label="Edit Theme">
           <vaadin-icon src="./assets/lucide-icons/palette.svg"></vaadin-icon>
+          <vaadin-tooltip slot="tooltip" text="Edit Theme"></vaadin-tooltip>
         </vaadin-button>
+        <vaadin-popover
+          for="theme-editor-btn"
+          position="top-start"
+          width="320px"
+          style="--aura-base-size: 12; --aura-base-font-size: 13"
+          theme
```

Comment:
Let me reiterate that I consider this empty `theme` attribute a hack which we should remove 🙂 
The fact that popover looks differently with and without empty `theme` is quite confusing.